Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -39,56 +46,60 @@ export default function useAndroidRippleForView(
     nativeBackgroundAndroid: NativeBackgroundProp,
   |}>,
 |}> {
+  const isSupported = Platform.OS === 'android' && Platform.Version >= 21;
+  if (!isSupported || !rippleConfig) {
+    return null;
+  }
   return useMemo(() => {
```

Comment:
> React Hook "useMemo" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return? (react-hooks/rules-of-hooks)

Reply:
hi, sorry for this, I didn't have properly functioning eslint ide integration so I didn't see those. There are no warnings now, but I'm not sure about the elegance of the solution. Please take a look, thanks.